R/fit.R

```diff
@@ -899,6 +926,9 @@ CmdStanMLE <- R6::R6Class(
   private = list(
     # inherits draws_ and metadata_ slots from CmdStanFit
     read_csv_ = function() {
+      if (!length(self$output_files(include_failed = FALSE))) {
+        stop("Optimization failed. Unable to retrieve the draws and metadata.")
```

Comment:
This message also mentions metadata but the ones for read_csv_ for MCMC and VB just mention draws. Maybe don't mention metadata here? Or add it to the other ones. I'm fine either way.

Reply:
I removed metadata, I agree its best for consistency.

Reply:
Thanks. Did you push that commit?

Reply:
Not yet. Need to add the test model and a few more tests for that model.

Reply:
Oh ok, no rush!